service/integration/open_pbs/README.rst

```diff
@@ -1,20 +1,28 @@
-geopm_open_pbs_power_limit(7) -- OpenPBS hook for providing job-level power limits
-==================================================================================
+geopm_open_pbs_power_limit(7) -- OpenPBS hooks for providing job-level power limits
+===================================================================================
 
 Overview
 --------
 
 The GEOPM OpenPBS power limit hook extends OpenPBS's functionality to allow
-the user to request a job-level power limit at the time of job submission. Via
-a command-line option, the user can specify a per-node power limit to be
+the user to request a job-level power limit through a ``geopm-job-power-limit``
+resource at the time of job submission, and to enforce cluster-level power
+limits through that mechanism. Via a command-line option, the user can specify
+a per-node power limit through a ``geopm-node-power-limit`` resource to be
 applied to every node that is part of the job. The power limit is applied just
 before the job runs, as part of the job's prologue, and is restored upon job
-completion, as part of the job's epilogue.
+completion, as part of the job's epilogue. The scheduler can be configured
+with constraints over those power limits across a cluster in order to enforce a
+cluster power limit. The GEOPM OpenPBS hook enables automatic configuration of
+job power limits to increase utilization of the available cluster power.
+
+.. image:: geopm_power_resources.svg
+     :width: 600
+     :alt: Block diagram of the relationship between geopm node and job power resources
 
 Save-restore Mechanism
 ----------------------
-
-The hook will keep track of the original power limit setting and restore it at
+The hook keeps track of the original power limit setting and restores it at
 the end of the job (for each node that's part of the job). To achieve this,
 the hook needs to be configured to run during the job prologue and epilogue
 events (see the Installation section for more details).
```

Comment:
Below:

`If the job is unexpectedly interrupted before the hook is able to restore the power limit setting, the power limit will be rectified the next time the hook is triggered on affected nodes. To accomplish this, the hook saves the original setting to a file before applying any changes. Typically, the hook will access the stored setting for the restore operation during the epilogue. During the prologue of a new job, if a new power limit is not requested and a save file is present, the hook will restore the saved power limit so that the new job is not affected by prior power limit settings.`

Question: Is the save file deleted following a successful restore? Might be worth stating explicitly if so

Reply:
It is. I'll add a note.